result/yoda.js

```diff
@@ -0,0 +1,16 @@
+
+/*eslint no-multi-spaces: "error"*/
+
+var a =  1;
+
+if(foo   === "bar") {}
+if(foo   === "bar") {}
+if(foo   === "bar") {}
+
+a <<  b
+a <<  b
+
+var arr = [1,  2];
```

Comment:
__[development][eslint]__
* Error - 'arr' is defined but never used (no-unused-vars)
* Error - Multiple spaces found before '2'. (no-multi-spaces)